bitnami/redis-cluster/templates/NOTES.txt

```diff
@@ -0,0 +1,57 @@
+** Please be patient while the chart is being deployed **
+
+{{ if .Values.usePassword }}
+To get your password run:
+
+    export REDIS_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "redis.secretName" . }} -o jsonpath="{.data.redis-password}" | base64 --decode)
+{{- end }}
+
+{{- if .Values.cluster.externalAccess.enabled }}
+
+To connect to your Redis server from outside the cluster check the following information:
+
+{{- if contains "LoadBalancer" .Values.cluster.externalAccess.service.type }}
```

Comment:
I still do not see if it makes sense having another externalAccess service type. If we don't support any other type apart from LoadBalancer then I wouldn't allow any other type.